### PR TITLE
Version 1.0.4 (Public Release)

### PR DESCRIPTION
**Pull request by aninda-ghosh**
- Added configuration and evaluation script for Full FTW and CC-BY Countries training.
- Added tensorboard for monitoring training progress.
- Added torchgeo 0.7 version information in the `upcoming features` section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -267,6 +272,30 @@ If training has been interrupted or if you wish to fine-tune a pre-trained model
 ftw model fit --config configs/example_config.yaml --ckpt_path <Checkpoint File Path>
 ```
 
+### VIsuaizing training process
```
Comment (cholmes) [inline README.md:275]:
```suggestion
### Visuaizing training process
```

File: README.md [hunk 2]
```diff
@@ -326,6 +355,17 @@ python run_experiments.py
...
 
+## Inference
+
+We provide two scripts that allow users to run models that have been pre-trained on FTW on any temporal pair of S2 images. First, you need to train a model (or download a pre-trained model), we provide an example pre-trained model in the [Releases](https://github.com/fieldsoftheworld/ftw-baselines/releases) list. Second, you need to concatenate the bands of two aligned Sentinel-2 scenes that show your area of interest in two seasons (e.g. planting and harvesting seasons) in the following order: B04_t1, BO3_t1, BO2_t1, B08_t1, B04_t2, BO3_t2, BO2_t2, B08_t2 (t1 and t2 represent two different points in time). The `download_imagery.py` script does this automatically given two STAC items. The Microsoft [Planetary Computer Explorer](https://planetarycomputer.microsoft.com/explore?d=sentinel-2-l2a) is a convinient tool for finding relevant scenes and their corresponding STAC items. Finally, `inference.py` is a script that will run a given model on overlapping patches of input imagery (i.e. the output of `download_imagery.py`) and stitch the results together in GeoTIFF format.
```
Comment (cholmes) [inline README.md:360]:
```suggestion
We provide two scripts that allow users to run models that have been pre-trained on FTW on any temporal pair of S2 images. First, you need a trained model - either download a pre-trained model (we provide an example pre-trained model in the [Releases](https://github.com/fieldsoftheworld/ftw-baselines/releases) list), or train your own model as explained in the [Training](#training) section. Second, you need to concatenate the bands of two aligned Sentinel-2 scenes that show your area of interest in two seasons (e.g. planting and harvesting seasons) in the following order: B04_t1, BO3_t1, BO2_t1, B08_t1, B04_t2, BO3_t2, BO2_t2, B08_t2 (t1 and t2 represent two different points in time). The `download_imagery.py` script does this automatically given two STAC items. The Microsoft [Planetary Computer Explorer](https://planetarycomputer.microsoft.com/explore?d=sentinel-2-l2a) is a convenient tool for finding relevant scenes and their corresponding STAC items. Finally, `inference.py` is a script that will run a given model on overlapping patches of input imagery (i.e. the output of `download_imagery.py`) and stitch the results together in GeoTIFF format.
```

Made some edit suggestions for the language.

**Review by cholmes — APPROVED** [review]
Looks good! Put a couple minor suggestions.

I do still want to get the latest two scripts into the CLI, so the instructions are all consistent with the `ftw` command-line. But good to have this better documentation of what we have.

**Comment by aninda-ghosh** [comment]
By integrating the latest scripts you mean to say the download imagery and large-scale inferencing script? @cholmes

**Comment by aninda-ghosh** [comment]
I will work on integrating these scripts in the next PR. Closing this PR for now since this PR was for a different purpose.

**Comment by cholmes** [comment]
> By integrating the latest scripts you mean to say the download imagery and large-scale inferencing script?

Yeah, exactly.

File: README.md (hunk 1 above)
Comment (calebrob6) [reply README.md:275]:
```suggestion
### Visualizing training process
```

File: README.md [hunk 3]
```diff
@@ -301,7 +330,7 @@ Options:
 Using FTW cli commands to test the model, you can pass specific options, such as selecting the GPUs, providing checkpoints, specifying countries for testing, and postprocessing results:
 
 ```bash
-ftw model test --gpu 0 --checkpoint_fn logs/path_to_model/checkpoints/last.ckpt --countries denmark finland --postprocess --output_fn results.csv
+ftw model test --gpu 0 --root_dir /path/to/dataset --checkpoint_fn logs/path_to_model/checkpoints/last.ckpt --countries country_to_test_on --output_fn results.csv
```
Comment (calebrob6) [inline README.md:333]:
```suggestion
ftw model test --gpu 0 --root_dir /path/to/dataset --checkpoint_fn logs/<PATH TO MODEL>/checkpoints/last.ckpt --countries <COUNTRY TO TEST ON> --output_fn results.csv
```

**Pull request merged**